tests/test_H0.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import pytest
+import os
+from datetime import datetime, timedelta
+
+from RadClass.RadClass import RadClass
+from RadClass.H0 import H0
+import tests.test_data as test_data
+
+# initialize sample data
+start_date = datetime(2019, 2, 2)
+delta = timedelta(seconds=1)
+timestamps = np.arange(start_date,
+                       start_date + (test_data.timesteps * delta),
+                       delta).astype('datetime64[s]').astype('float64')
+
+live = np.full((len(timestamps),), test_data.livetime)
+sample_val = 1.0
+spectra = np.full((len(timestamps), test_data.energy_bins),
+                  np.full((1, test_data.energy_bins), sample_val))
+# setting up for rejected null hypothesis
+spectra[test_data.timesteps//2:] = 100.0
```

Comment:
I think we should define this index as a variable and reuse it:
`rejected_H0_time = test_data.time_steps//2`

Reply:
I see, added ✔️